src/client/refresh/gl3/gl3_main.c

```diff
@@ -530,6 +536,32 @@ GL3_Init(void)
 		R_Printf(PRINT_ALL, " - OpenGL Debug Output: Not Supported\n");
 	}
 
+	gl3config.useBigVBO = false;
+	if(gl3_usebigvbo->value == 1.0f)
+	{
+		R_Printf(PRINT_ALL, "Enabling useBigVBO workaround because gl3_usebigvbo = 1\n");
+		gl3config.useBigVBO = true;
+	}
+	else if(gl3_usebigvbo->value == -1.0f)
+	{
+		// enable for AMDs proprietary Windows and Linux drivers
+		// TODO: should we match a version number? does the workaround
+		//       slow down legacy drivers that work fine without it?
```

Comment:
I need to test this with the AMD legacy driver on my Radeon HD 6850.
I think the GL3 renderer worked fine there without this workaround, but the real question is if it gets slower with the workaround enabled (which would mean we'd have to check the driver version number)

Reply:
.. of course it turns out that the legacy driver doesn't like this code and runs much smoother without the workaround :-/

(For the record, it's really a Radeon HD 6**9**50 and it's  the "Radeon Software Crimson Edition Beta" driver version 16.2.1 Beta, on Win7 x64. This is the latest driver AMD offers for that GPU.)

Reply:
matching the version now..